lms/djangoapps/verified_track_content/models.py

```diff
@@ -49,8 +53,9 @@ def move_to_verified_cohort(sender, instance, **kwargs):  # pylint: disable=unus
                 sync_cohort_with_mode.apply_async(kwargs=args, countdown=300)
             else:
                 log.error(
-                    "Automatic verified cohorting enabled for course '%s', but course does not have a verified cohort",
-                    course_key
+                    "Automatic verified cohorting enabled for course '%s', but cohort named %s does not exist.",
```

Comment:
Please put single quotes around the name (consistent with the rest of the log messages).